atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -64,10 +64,8 @@
 class AddLabelsPlugin(PreBuildPlugin):
     key = "add_labels_in_dockerfile"
     is_allowed_to_fail = False
-    rewrite_whitelist = ['release', 'version']
 
-    def __init__(self, tasker, workflow, labels,
-                 dont_overwrite=("Architecture", "architecture"),
+    def __init__(self, tasker, workflow, labels, dont_overwrite=None,
```

Comment:
Nitpick, but if you want to simplify this, you can use `dont_overwrite=tuple()` to default to an empty tuple in method signature instead of in the method body.